client/components/header-component.js

```diff
@@ -1,14 +1,20 @@
 const HeaderComponent = {
-  template: `<div class="header">
-  <a class="link" href="/">
-    <img alt="HYF_connect logo" src="/public/assets/logo_pc_full.png" width="90"/>
-  </a>
-  <a class="link" href="/pages/projects/projects.html">projects</a>
-  <a class="link" href="/pages/members/members.html">members</a>
-  <a class="link" href="/pages/about-us/about-us.html">about us</a>
-  <a class="link" href="/pages/contact-us/contact-us.html">contact us</a>
-  <button>Log In</button>
-</div>`
+  template: `<section class="header">
+  <div class="header-container">
+      <div class="header-left">
+      <a href="https:" class="logo">
+      <img src="/client/public/assets/logo_pc_full.png" alt="logo" width="100" height="80"/></a>
```

Comment:
Normally when running the server using nom run start the url path should not contain "/client" as the client is the only folder that is served by the backend